build/Makefile

```diff
@@ -421,8 +426,10 @@ endif
 
 build-controller-image-amd64: $(ensure-build-image) build-controller-binary build-licenses build-required-src-dist
 	docker build $(agones_path)/cmd/controller/ --tag=$(controller_amd64_tag) $(DOCKER_BUILD_ARGS)
+# creating docker builder and then using that builder to build controller image in buildx command
 build-controller-image-arm64: $(ensure-build-image) build-controller-binary
-	docker buildx build --platform linux/arm64 --build-arg ARCH=arm64 $(agones_path)/cmd/controller/ --tag=$(controller_arm64_tag) $(DOCKER_BUILD_ARGS)
+	$(MAKE) create-arm64-builder 
+	docker buildx build --platform linux/arm64 --builder $(BUILDX_ARM64_BUILDER) --build-arg ARCH=arm64 $(agones_path)/cmd/controller/ --tag=$(controller_arm64_tag) $(DOCKER_BUILD_ARGS)
```

Comment:
So we're only using this builder for arm64 images? Just for my own education, is there a reason we don't need it for the amd64 images?

Reply:
We are actually building multi-arch images for which builder is required in buildx command.

Reply:
We were chatting about this - just as a note - we are using the multi-arch build process to build out a single arch image (linux/arm64) -- but the important note is that for a M1 mac, there is no reason to use a builder on `docker build`  to build images.